english/play-some-cricket/playsomecricket.md

```diff
@@ -0,0 +1,6 @@
+[Checkout most rare and funny moments incricket history.](https://youtu.be/fzqh6_90akI)
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.